assets/js/src/newsletters/automatic_emails.jsx

```diff
@@ -0,0 +1,283 @@
+import React from 'react';
+import _ from 'underscore';
+import Hooks from 'wp-js-hooks';
+import MailPoet from 'mailpoet';
+import AutomaticEmailEventsList from 'newsletters/types/automatic_emails/events_list.jsx';
+import EventsConditions from 'newsletters/automatic_emails/events_conditions.jsx';
+import AutomaticEmailsBreadcrumb from 'newsletters/types/automatic_emails/breadcrumb.jsx';
+import SendEventConditions from 'newsletters/automatic_emails/send_event_conditions.jsx';
+import Listings from 'newsletters/automatic_emails/listings.jsx';
+
+const emails = window.mailpoet_premium_automatic_emails || [];
+const newslettersContainer = document.getElementById('newsletters_container');
+
+if (newslettersContainer && !_.isEmpty(emails)) {
+  const addEmails = (types, that) => {
+    // remove automatic emails declared in Free version if they are declared in Premium
+    const existingTypes = _.reject(types, (type) => _.has(emails, type.slug));
+    const newTypes = _.map(emails, (email) => {
+      const updatedEmail = email;
+      const onClick = _.partial(that.setupNewsletter, email.slug);
+
+      updatedEmail.action = (() => (
+        <div>
+          <span
+            className="button button-primary"
+            onClick={onClick}
+            onKeyDown={(event) => {
+              if ((['keydown', 'keypress'].includes(event.type) && ['Enter', ' '].includes(event.key))
+              ) {
+                event.preventDefault();
+                onClick();
+              }
+            }}
+            role="button"
+            data-automation-id="create_woocommerce"
+            tabIndex={0}
+          >
+            {email.actionButtonTitle || MailPoet.I18n.t('setUp')}
+          </span>
+        </div>
+      ))();
+
+      return updatedEmail;
+    });
+
+    return [...existingTypes, ...newTypes];
+  };
+
+  const addEmailsRoutes = (routes) => {
+    // remove routes declared in Free version if they are declared in Premium
```

Comment:
This comment suggests we don't need this piece of code.

Reply:
Same here.